_module_templates/wrapper_module_template.md

```diff
@@ -0,0 +1,475 @@
+<!--
+
+author:   Your Name
+email:    email@chop.edu
+version:  0.0.0
+module_type: wrapper
+module_template_version: 1.0.0
+language: en
+narrator: UK English Female
+
+title: Module Title
+
+comment:  This is a short, focused description of the module.
+
+long_description: This is a longer description, which should be understandable for a lay audience. It will print under "Is this module right for me?" in the overview.
+
+estimated_time: This is rough guess of how long it might take a learner to work through the module. It will print under "Estimated time to completion" in the overview
+
+r_file: this\_r\_module
+
+@prerequisites
+List any skills and knowledge needed to do this module here. When available, include links to resources, especially other modules we've made (to show learners where this falls within our catalog).
+
+* one skill we have [another module for, linked here](https://education.arcus.chop.edu)
+* some familiarity with [a topic](https://education.arcus.chop.edu)
+* understanding of [one thing](https://education.arcus.chop.edu) and [another](https://education.arcus.chop.edu)
+
+If relevant, you can include recommendations for somewhere else to start if the learner doesn't have these prereqs. For example: If you are brand new to R or python (or want a refresher) consider starting with [Intro to R](link) or [Intro to python](link) first and then coming back here.
+@end
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- identify key elements
+- create a product
+- do a task
+- articulate the rationale for something
+@end
+
+
+import: https://raw.githubusercontent.com/arcus/education_modules/template_test/a_sample_module_template/module_macros.md
+-->
```

Comment:
Given the fact that a lot of these values are ones that appear in other module templates, are there any ideas for reducing the number of places that are required to be updated any time we have a change? does that seem like a worthwhile enough thing to think about given effort-value trade-offs? 

I'm thinking especially about instances where we may want to add a YAML field to all of the module types, or change one that already exists in all of them (e.g., currently with estimated_time) -- how reliable do we think we'll be able to be at ensuring they stay consistent across all the templates?

Reply:
I feel optimistic about keeping three templates up to date, at least for now. We've done a nice job of improving processes in an agile way for this so far -- maybe if we start to notice the templates are becoming a pain to maintain, we can optimize then.
If we did want to remove copied text across templates, I think the way to do that would be to have e.g. a bash script to generate the template files from standardized bits of text (so we'd have a txt file of the YAML items that are the same across all templates, and then we'd have files of the YAML items that are specific to each template, and we'd have a script combine them appropriately when generating the templates themselves). Right now that feels like more work to create and maintain than time/errors it would save? Maybe you have a better idea, though.